.circleci/test-content-server.sh

```diff
@@ -49,7 +49,7 @@ if grep -e "$MODULE" -e 'all' $DIR/../packages/test.list; then
   # ensure email-service is ready
   check 127.0.0.1:8001/__heartbeat__
   cd packages/fxa-content-server
-  mozdownload --version 58.0 --destination firefox.tar.bz2
+  mozdownload --version 67.0 --destination firefox.tar.bz2
```

Comment:
@shane-tomlinson Is there some reason we wouldn't use `latest` here, instead of a specific version number?

Reply:
The biggest reason is that sometimes digdug/theintern do not work with the latest and greatest.